AUTHORS

```diff
@@ -0,0 +1 @@
+../d2/mods/d2/AUTHORS
```

Comment:
This path is funky.
I _suspect_ you want `./mods/d2/AUTHORS` if you're going to put a path in this file.